src/test/reverse.test.js

```diff
@@ -0,0 +1,5 @@
+import reverse from '../jsfiles/reverse';
+
+test('Return reversed string', () => {
+  expect(reverse('backwards')).toBe('sdrawkcab');
```

Comment:
- [ ] [For next projects] When you write tests for real remember to add more than one case for each method, otherwise it could only be passing with one specific input and you want to make sure that doesn't happen.